docs/excel/performance.md

```diff
@@ -157,6 +157,34 @@ Excel.run(async (ctx) => {
 > [!NOTE]
 > You can conveniently convert a Table object to a Range object by using the [Table.convertToRange()](https://docs.microsoft.com/javascript/api/excel/excel.table#converttorange--) method.
 
+## Untrack unneeded ranges
```

Comment:
Should this be singular?  (i.e., does Range.untrack() only untrack the range for which it is called?)

-> "Untrack a range"

Reply:
Hmm, it's true `untrack()` only applies to a single range, but the problem we're solving is tracking too many ranges. I wanted the header to imply scope. You've got too many ranges; what now? There's kind of the same implication in the note about ClientRequestContext; sure you can untrack a single table, but you don't need to.

Reply:
Good point, that makes sense. Carry on as you were, then :)